### ISSUE TITLE
[Feature Request] Snake-based logo

### ISSUE THREAD

**Issue by Zeebrow** [facebookresearch/hydra#1508]
# 🚀 Feature Request

Draw a snake-based logo for this project.

## Motivation

The README.md depicts a three-headed DRAGON as the representative for the Hydra Python package. I cannot stand idly by while the perfect opportunity exists for a snake-based logo.

## Pitch
How is one expected to sell the idea of a Python package, without an apodal, apterous, reptilian mascot?

I was brought here whilst searching for alternatives to configparser. I immediately scroll to the README and am greeted by this medieval abstraction of a gatekeeper. I have yet to look at the sourcecode, out of fear of reprisal from this demon lizard. Since I can't look at the sourcecode, I'm honetly not sure this is even a Python package. 

In other words, the aforementioned barrier to entry makes this package is utterly useless.

## Additional context
I will see what I can whip up in MS paint this weekend.

**Comment by omry** [comment]
I am sorry out friendly non-snaky Hydra is making you uncomfortable. You can be sure that he has nothing but good intentions.
At this time there are no plans to fire our Hydra, definitely not in favor of a snaky based Hydra. 
I would be happy to see what you come up with (Don't expect the logo to actually change :) ).

**Comment by Zeebrow** [comment]
I'm shocked and appalled. Closing issue with unfathomable contempt.

(/s just incase)

**Comment by omry** [comment]
I hope your irrational hatred for our blue and helpful multi headed friend will prevent you from getting his help.
